ironfish-rust-nodejs/src/frost.rs

```diff
@@ -52,3 +54,60 @@ pub fn round_two(
 
     Ok(bytes_to_hex(&signature_share.serialize()))
 }
+
+#[napi]
+pub struct ParticipantSecret {
+    secret: Secret,
+}
+
+#[napi]
+impl ParticipantSecret {
+    // TODO(hughy): implement Secret ser/de
+    #[napi(constructor)]
+    pub fn random() -> ParticipantSecret {
+        let secret = Secret::random(thread_rng());
+
+        ParticipantSecret { secret }
+    }
+
+    #[napi]
+    pub fn to_identity(&self) -> Result<ParticipantIdentity> {
+        let identity = self.secret.to_identity();
+
+        Ok(ParticipantIdentity { identity })
+    }
+}
+
+#[napi]
+pub struct ParticipantIdentity {
+    identity: Identity,
+}
+
```

Comment:
Will we need an `Identifer` object at all in these bindings? What about when we are doing encryption/validation via the public keys (identifier). Maybe we don't, we also can probably just wait until we need it.